test/client/views/feedBody.js

```diff
@@ -8,17 +12,40 @@ var FeedBody = require('../../../source/js/views/feedBody');
 var internals = {};
 
 
-describe('FeedBody', function () {
+describe('FeedBodyView', function () {
 
-    describe('when a request is add to the collection', function () {
+    var fakeModel, view;
 
-        it('adds a request to the feed body');
+    beforeEach(function() {
+
+        fakeCollection = new Backbone.Collection();
+        view = new FeedBodyView({ collection: fakeCollection });
+        view.render();
+    });
+
+    describe('when a request is added to the collection', function () {
+
+        it('adds a request to the feed body', function() {
+
+            var request = new RequestView();
+
+            view._addRequest(request);
```

Comment:
It's best not to call private methods from tests. The expectation is that someone can refactor all of the private methods of an object, and non of the tests should need to be updated. This can only be accomplished if the tests are only using public interfaces. You'll need to figure out how to exercise the test though public interfaces only. For example, instead of calling `_addRequest`, you can pass in a collection to the view and then add the request to the collection. Instead of checking your expectation against `view._requestViews`, you can render the view and check against the rendered markup.